lib/livyClient/lib/client.js

```diff
@@ -11,45 +11,82 @@ const GET_BACKOFF_PARAMS = { min: 100, max: 200, maxAttempts: 300,
 const POST_BACKOFF_PARAMS = { min: 1000, max: 3000, maxAttempts: 20,
     factor: 1.1 };
 
+function inputCheck(batchOrSession, id, callback) {
```

Comment:
move this into the class as a method and call _inputCheck to make clearer that internal function